### ISSUE TITLE
Multiple problems when building under Linux

### ISSUE THREAD

**Issue by SoftCreatR** [renpy/renpy#2097]
I'm relatively new to Ren'Py (started 2 days ago to use it), so i'm sorry, if i'm missing something.

I've got a game, that compiles without problems under Windows. However, i would like to automate the build process on a linux machine, so i tried to build it there:

`./renpy.sh launcher distribute --no-update --dest /build /game-source`

The result looks promising:

```Scanning project files...
Archiving files... - 60 of 60
Archiving files... - 59 of 59
Archiving files... - 2 of 2
Archiving files... - 2989 of 2989
Archiving files... - 1456 of 1456
Archiving files... - 11 of 11
Archiving files... - 112 of 112
Scanning Ren'Py files...
Writing the pc zip package. - 1099 of 1099
Writing the linux tar.bz2 package. - 970 of 970
Writing the mac app-zip package. - 836 of 836
Writing the win zip package. - 811 of 811
Writing the market zip package. - 1473 of 1473
Writing the web zip package. - 239 of 239
All packages have been built.

Due to the presence of permission information, unpacking and repacking the Linux and Macintosh distributions on Windows is not supported.
```

But when running on my desktop PC, i'm experiencing multiple problems.

1. The main menu music doesn't play
2. The `win` version crashes during the playback of the sample sound
3. Both, the `win` version and the `pc` version reporting `Unsupported image format` when loading a specific image file

When performing a comparison between the builds that have been created under Windows 10 and the builds that have been created under Linux (Debian 10, to be exact), there are several differences (left: Built under Linux, right: Built under Windows):

![image](https://user-images.githubusercontent.com/81188/72787860-026d4100-3c31-11ea-8301-5ba7126193d9.png)

Furthermore, there are obvious differences in the build sizes:

![image](https://user-images.githubusercontent.com/81188/72788067-6bed4f80-3c31-11ea-8109-b6175bfa15f3.png)

Just as reminder: When built under Windows, the game works flawlessly without any errors or problems.

So there must be something different between the Ren'Py SDK for Windows and the SDK that is provided as tar.bz2. Or is it another issue like missing packages on the server?

The game creates a log.txt during startup. I've compared both, but except different loading times, everything else matches:

[log.txt](https://github.com/renpy/renpy/files/4090028/log.txt)
[log.txt](https://github.com/renpy/renpy/files/4090029/log.txt)

**Comment by SoftCreatR** [comment]
Quick follow up: I've uploaded the SDK for Windows on the server and started another build. The results are the same, so the problem must be somewhere else. I also tried Ubuntu, but got the same results.

**Comment by renpytom** [comment]
Hopefully this is fixed in 7.4. I don't have any idea what could cause this,
